lib/PodLogs.py

```diff
@@ -323,8 +323,11 @@ def is_running(self, pod):
         # implemented here is because it seemed that pods could go from 'Success' back into
         # 'Running' or 'Pending'.  So we're just doing a list of the pods, and then return
         # true if the pod is in the list.
-        pods = self.list_namespaced_pod()
-        job_pod_names = [p for p in pods.items if pod == p.metadata.name]
+        pods = self.list_namespaced_pod(pod)
```

Comment:
It seems odd that we are passing in pod and not using it in the function.

Reply:
We use it just to mention the pod that couldn't be followed:
```install_logger.debug(f"Could not list pods.  Giving up on pod {pod}.")```